pom.xml

```diff
@@ -17,24 +17,23 @@
         <maven.checkstyle.plugin.configLocation>
             https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
-        <spring.version>5.2.2.RELEASE</spring.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
-            <version>${spring.version}</version>
+            <version>5.3.14</version>
```

Comment:
why did u remove this ?

Reply:
I'm also remove  <spring.version>5.2.2.RELEASE</spring.version>. On this version I have got an exception and 5.3.14 maven could not download, that;s why I put separatly version 5.3.14 in 3 dependency

Reply:
that's pretty strange 0_o